### PR TITLE
Adds proper suicide messages to like a lot of stuff

### PR DESCRIPTION
**Pull request by dannno**
Now you can shoot yourself in your stupid face you chump

### CODE REVIEW

**Comment by Chocobro** [comment]
suicide is never the answer

(:+1)

**Comment by Kromgar** [comment]
Does it cause brute damage to the face too?

**Comment by GunHog** [comment]
Projectile weapons spawn a gib or two, to simulate blowing your own brains out.

**Comment by Incoming5643** [comment]
You can use this to kill yourself with physically harmless wands, but at the same time I think that's funny enough to not be a problem

**Comment by Miauw62** [comment]
>no suicide_act() for banana

0/10

**Comment by dannno** [comment]
you got it boss

**Comment by dannno** [comment]
thats the ish that i like

**Comment by dannno** [comment]
do I need a changelog for this

**Comment by Miauw62** [comment]
if you want to.

**Comment by Kraseo** [comment]
suicide by banana should have some sleep()s on them so the messages aren't instant dannonino

File: html/changelogs/Danno-Suicide-7496.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: Dannno
+
+delete-after: True
+  - rscadd: "Added specific suicide messages to guns of all types."
+  - rscadd: "Added special suicide messages to bananas :^)"
```
Comment (Iamgoofball) [inline html/changelogs/Danno-Suicide-7496.yml]: :^) has infiltrated the code

**Comment by JJRcop** [comment]
In my more suicides pr (which was actually pretty chelpy in retrospect), I made guns actually blow your brains out.
#1343

**Comment by dannno** [comment]
do you mind if I steal that and maybe some other shit from your PR

**Comment by JJRcop** [comment]
That's fine, I was gonna try to make it again some time soon with less looney-toons wacky crazy suicides and just make things that make sense, but I never got around to it.

I remember it caused runtimes, I didn't transfer the mind properly and blah blah so..

File: code/game/objects/items/weapons/handcuffs.dm [hunk 2]
```diff
@@ -153,6 +153,11 @@
 	var/armed = 0
 
 /obj/item/weapon/restraints/legcuffs/beartrap/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is sticking \his head in the [src.name]! It looks like \he's trying to commit suicide.</span>")
+	playsound(loc, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
+	return (BRUTELOSS)
+
+/obj/item/weapon/restraints/legcuffs/beartrap/suicide_act(mob/user)
```
Comment (Alek2ander) [inline code/game/objects/items/weapons/handcuffs.dm]: What's going on here?

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 3]
```diff
@@ -110,6 +110,10 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/smoketime = 300
 	var/chem_volume = 30
 
+/obj/item/clothing/mask/cigarette/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is huffing the [src.name] is quickly as they can! It looks like \he's trying to give \himself cancer.</span>")
```
Comment (Alek2ander) [inline code/game/objects/items/weapons/cigs_lighters.dm]: >is quickly as they can

File: code/game/objects/items/weapons/storage/book.dm [hunk 4]
```diff
@@ -18,6 +18,10 @@
 	var/mob/affecting = null
 	var/deity_name = "Christ"
 
+/obj/item/weapon/storage/book/bible/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is offering /himself to \his deity! It looks like \he's trying to commit suicide.</span>")
```
Comment (Alek2ander) [inline code/game/objects/items/weapons/storage/book.dm]: > /himself
Also, the deity usually has a name too, muh immurshins.

File: code/game/objects/items/weapons/handcuffs.dm (hunk 2 above)
Comment (Dorsisdwarf) [inline code/game/objects/items/weapons/handcuffs.dm]: Grayshirt McDepressed is sticking his head in the beartrap! It looks like he's trying to commit suicide.

Comment (JJRcop) [inline code/game/objects/items/weapons/handcuffs.dm]: The old suicide_act is still there.

Comment (dannno) [inline code/game/objects/items/weapons/handcuffs.dm]: woops

File: code/game/objects/items/weapons/cigs_lighters.dm (hunk 3 above)
Comment (dannno) [inline code/game/objects/items/weapons/cigs_lighters.dm]: woops

File: code/game/objects/items/weapons/storage/book.dm (hunk 4 above)
Comment (dannno) [inline code/game/objects/items/weapons/storage/book.dm]: i'll look into making it say the deity name

**Comment by dannno** [comment]
why failed

**Comment by JJRcop** [comment]
Someone messed up z2

**Comment by dannno** [comment]
wat

**Comment by JJRcop** [comment]
It should be fine now

**Comment by dannno** [comment]
hg if ur merging stuff merge this 2

**Comment by dannno** [comment]
meeeeeeeerg

**Comment by JJRcop** [comment]
[quack](http://en.wikipedia.org/wiki/Mergus)

**Pull request merged**